app/assets/javascripts/components/verify-button.js

```diff
@@ -26,7 +26,8 @@ ManageIQ.angular.app.component('verifyButton', {
           {target: '.validate_button:visible'},
           vm.valtype,
           true,
-          vm.findScope(), vm.validateUrl
+          vm.findScope,
```

Comment:
I'm really not comfortable with the `findScope` function written in one of the earlier related PRs - https://github.com/ManageIQ/manageiq-ui-classic/pull/2017/commits/1a60cb4ecf6b10836351cfaccb4fb24a58a251d6

```javascript
vm.findScope = function() {
  var parentScope = $scope;
  while (! parentScope.angularForm) {
        parentScope = parentScope.$parent;
   }
   return parentScope.angularForm;
 };
```

I wonder if there is a better alternative.

Reply:
Yep i don't like it either. This component is actually obsolete and will not be used. After a short discussion with @himdel we decided not to use this component/partial at all and use the miq-button instead and let the controller to handle the validation request because the service used in current  code unfortunately relies on controller scope. So i will remove this component in near future.